core/VigilanceSettings.js

```diff
@@ -5,13 +5,14 @@ const localDir = "./config/ChatTriggers/modules/"
 
 /**
  * - Converts a vigilance setting into a [JSON] format for this settings gui
- * @param {Object} obj 
- * @param {String} moduleName 
+ * @param {Object} obj Your Vigilance settings class
+ * @param {String} moduleName "Amaterasu"
+ * @param {String} path "/data/defaultSettings"
  * @param {Boolean} overWrite Whether it should overwrite the old file if it exists or not (false by default)
  * @returns 
  */
-export const convertToJSON = (instance, moduleName, overWrite = false, moduleToConvert = null) => {
-    if (FileLib.exists(`${localDir}${moduleName}/data/defaultSettings.json`) && !overWrite) return
+export const convertToJSON = (instance, moduleName, path, overWrite = false, moduleToConvert = null) => {
```

Comment:
shouldn't really matter since it's a 1 time type of thing you can just copy and paste it to the correct path